db/change_cache.go

```diff
@@ -472,7 +472,7 @@ func (c *changeCache) DocChanged(event sgbucket.FeedEvent) {
 
 	// If the doc update wasted any sequences due to conflicts, add empty entries for them:
 	for _, seq := range syncData.UnusedSequences {
-		base.Infof(base.KeyCache, "Received unused #%d in _sync.unused_sequences property for (%q / %q)", seq, base.UD(docID), syncData.CurrentRev)
+		base.Infof(base.KeyCache, "Received unused #%d in "+base.SyncPropertyName+".unused_sequences property for (%q / %q)", seq, base.UD(docID), syncData.CurrentRev)
```

Comment:
For this case and the next one, I think the _sync could be removed altogether.
```suggestion
		base.Infof(base.KeyCache, "Received unused #%d in unused_sequences property for (%q / %q)", seq, base.UD(docID), syncData.CurrentRev)
```